src/js/calculator/home_care_client_fee/_translations.js

```diff
@@ -177,24 +172,19 @@ const translations = {
     en: 'If both spouses are clients of continuous and regular home care with a monthly fee, the fee is divided between them in accordance with their service and care plans.',
   },
   receipt_shopping_service_math_single: {
-    fi: '1 kauppatoimitus viikossa, kuukausihintaan on laskettu 4 toimitusta.',
-    sv: '1 butiksleverans per vecka, 4 leveranser ingår i månadspriset.',
-    en: '1 grocery delivery per week; the monthly fee includes 4 deliveries.',
+    fi: '1 kauppatoimitus viikossa, kuukausihintaan on laskettu 4 toimitusta.',
```

Comment:
Why are you removing the non-breaking-space characters here?

Reply:
Btw, I recommend installing a plugin to your IDE to highlight special characters like these to avoid accidental edits.
![image](https://github.com/City-of-Helsinki/drupal-hdbt/assets/1191667/37f8fa17-33c7-42f5-a990-fa87521b4721)

Reply:
Thanks for explaining this one! Returned the characters since they were really good idea!